curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657d4eb2785e0d268dbd2ad2.md

```diff
@@ -1,37 +1,41 @@
 ---
-id: 64b457bb4cd0e2013d839c92
-title: Step 48
+id: 657d4eb2785e0d268dbd2ad2
+title: Step 58
 challengeType: 0
-dashedName: step-48
+dashedName: step-58
 ---
 
 # --description--
 
-Inside the loop, use an `if` statement to check if the current element `radioButton` is checked using `.checked` property. If it is `true`, then reassign the `selectedValue` to `radioButton.value`.
+If the current radio button inside the loop has the checked attribute, then you will need to get its `value` and `id`.
 
-Then, inside the `if` statement, reassign the `achieved` variable to `radioButton.id`. In the end, add a `break` statement to exit the loop.
+Add an `if` statement inside your `for...loop`, with a condition of `radioButton.checked`. 
+
+Inside your `if` statement, reassign `radioButton.value` to `selectedValue` and reassign `radioButton.id` to `achieved`. 
```

Comment:
`assign` might be a better term here since these variables were declared without being initialized with a value:

```suggestion
Inside your `if` statement, assign `radioButton.value` to `selectedValue` and assign `radioButton.id` to `achieved`.
```